CNTK.Cpp.props

```diff
@@ -9,7 +9,7 @@
     <NvmlDll Condition="Exists('c:\local\nvsmi9\NVSMI\nvml.dll')">c:\local\nvsmi9\NVSMI\nvml.dll</NvmlDll>
 
     <HasOpenCv>false</HasOpenCv>
-    <HasOpenCv Condition="Exists('$(OPENCV_PATH)') Or Exists('$(OPENCV_PATH_V31)')">true</HasOpenCv>
+    <HasOpenCv Condition="Exists('$(OPENCV_PATH)') Or Exists('$(OPENCV_PATH_V31)') Or Exists('$(OPENCV_PATH_V34)')">true</HasOpenCv>
```

Comment:
Is this really needed in this PR? If not, I think we should undo this and leave only one version of OpenCV in there (the one that we have listed on our installation page).